pilot/cmd/pilot-agent/main.go

```diff
@@ -131,6 +133,8 @@ var (
 		"max number of message events for stackdriver")
 	pilotCertProvider = env.RegisterStringVar("PILOT_CERT_PROVIDER", "citadel",
 		"the provider of Pilot DNS certificate.").Get()
+	jwtPolicy = env.RegisterStringVar("JWT_POLICY", jwt.JWTPolicyThirdPartyJWT,
```

Comment:
this should match the values.yaml default (which I think should be 3p)

Reply:
Changed to match values.yaml default. In this PR, the JWT policy is configured to be first-party-jwt so that the first-party-jwt support can be tested on all unit tests/integration tests.

Reply:
why do we need this flag on both pilot server and pilot agent?

Reply:
hw are you going to reconcile the differences?

Reply:
This flag is needed on both pilot server and pilot agent because control plane and data plane need to know which type of JWT to use. The JWT provider is configured through the jwtProvider parameter at https://github.com/istio/istio/blob/master/manifests/global.yaml#L568. A prior PR on adding the first-party-jwt support is https://github.com/istio/istio/pull/20179.